neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
```

Comment:
I meant that for me, trunk_direction is vector(second_cross_section - first_cross_section), and initial_direction is vector(first_cross_section - soma.center).

If I understand correctly, both will be the same?